djangosaml2/views.py

```diff
@@ -787,14 +788,23 @@ def do_logout_service(self, request, data, binding):
         return HttpResponseBadRequest("No SAMLResponse or SAMLRequest parameter found")
 
 
-def finish_logout(request, response, next_page=None):
+def finish_logout(request, response):
     if getattr(settings, "SAML_IGNORE_LOGOUT_ERRORS", False) or (
         response and response.status_ok()
     ):
-        if not next_page:
-            next_page = getattr(settings, "LOGOUT_REDIRECT_URL", "/")
```

Comment:
This remove the LOGOUT_REDIRECT_URL, I think that this is a good feature indeed. can we enable it again?

Reply:
I think that also a review on the Docs is needed if something that's there have been removed in this PR

Reply:
indeed, I'll get back to you